README.md

```diff
@@ -46,15 +45,13 @@ git clone -b v17.9.1 --single-branch git@github.com:GEOS-ESM/GEOSldas.git
 To build the model in a single step, do the following from a head node:
 ```
 cd ./GEOSldas
-parallel_build.csh
+./parallel_build.csh -mil
 ```
 This checks out all the external repositories of the model (albeit only on the first run, [see subsection on mepo below](#mepo)!) and then builds and installs the model. 
 
-At **NCCS**, the default is to build GEOSldas on SLES12 (Skylake or Cascade Lake nodes); to build GEOSldas on SLES15 (Milan nodes), use `parallel_build.csh -mil`.
+The resulting model build is found in `build-SLES15/`, and the installation is found in `install-SLES15/`, with setup scripts like `ldas_setup` in `install-SLES15/bin`.
```

Comment:
I'm a bit confused by the build and install directory names, which may be outdated.  When I look at the build of the LDAS nightly tests, there's not "-SLES15" in the directory names:
```
discover36:~ ll /discover/nobackup/mathomp4/SystemTests/builds/LDAS_CONUS/CURRENT/GEOSldas/ |grep build
drwxr-xr-x 13 mathomp4 g0620    4096 Mar 10 23:07 build-Aggressive/
drwxr-xr-x 13 mathomp4 g0620    4096 Mar 10 22:40 build-Debug/
drwxr-xr-x 13 mathomp4 g0620    4096 Mar 10 20:23 build-Release/

```